### PR TITLE
Update indentation on json decorator example

### PR DESCRIPTION
**Pull request by bendews**
Small indentation change to fix the broken code block for the `json` decorator class on https://uplink.readthedocs.io/en/stable/decorators.html

Attention: @prkumar

### CODE REVIEW

File: uplink/decorators.py [hunk 1]
```diff
@@ -286,12 +286,12 @@ def update_user(self, **info: Body):
...
-    .. code-block:: python
+        .. code-block:: python
 
-        @json
-        @patch(/user")
-        def update_user(self, name: Field, email: Field("e-mail"):
-            \"""Update the current user.\"""
+            @json
+            @patch(/user")
+            def update_user(self, name: Field, email: Field("e-mail"):
```
Comment (prkumar) [inline uplink/decorators.py]: Whoops! Looks like I forgot to add a trailing parenthesis here 🤦‍♂️

**Comment by prkumar** [comment]
@bendews - Thanks for addressing this! 

Note that after your branch is merged into `master`, your changes will be reflected on https://uplink.readthedocs.io/en/latest/dev/decorators.html, which is a deferent URI than the `stable` documentation (which you linked in the description) because we're reorganizing docs prior to the `v0.6.0` release.

File: uplink/decorators.py (hunk 1 above)
Comment (bendews) [reply uplink/decorators.py]: Don't worry, I missed it as well even though I was triple checking! Good catch, I've added that to the PR 👍

**Comment by prkumar** [comment]
@bendews - Merging this into `master` now. Thanks again!

**Pull request merged**